Userland/Games/FlappyBug/Game.h

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2021, Mim Hufford <mim@hotmail.co.uk>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Optional.h>
+#include <AK/Random.h>
+#include <LibGUI/Application.h>
+#include <LibGUI/Painter.h>
+#include <LibGUI/Widget.h>
+#include <LibGfx/Bitmap.h>
+#include <LibGfx/Font.h>
+#include <LibGfx/StandardCursor.h>
+
+namespace FlappyBug {
+
+class Game final : public GUI::Widget {
+    C_OBJECT(Game);
+
+public:
+    static const int game_width = 560;
+    static const int game_height = 480;
+
+private:
+    Game();
+
+    virtual void paint_event(GUI::PaintEvent&) override;
+    virtual void keydown_event(GUI::KeyEvent&) override;
+    virtual void timer_event(Core::TimerEvent&) override;
+
+    void tick();
+    void reset();
+    void game_over();
+    bool ready_to_start() const;
+
+    struct Bug {
+        const float x { 50 };
+        const float radius { 16 };
+        const float starting_y { 200 };
+        const RefPtr<Gfx::Bitmap> falling_bitmap { Gfx::Bitmap::load_from_file("res/icons/flappybug/falling.png") };
+        const RefPtr<Gfx::Bitmap> flapping_bitmap { Gfx::Bitmap::load_from_file("res/icons/flappybug/flapping.png") };
```

Comment:
```suggestion
        const RefPtr<Gfx::Bitmap> falling_bitmap { Gfx::Bitmap::load_from_file("/res/icons/flappybug/falling.png") };
        const RefPtr<Gfx::Bitmap> flapping_bitmap { Gfx::Bitmap::load_from_file("/res/icons/flappybug/flapping.png") };
```

You're using a relative path here which means this only works when executing the game with a current working dir of `/`.

Reply:
Amazing how hard you can face palm when working in a whole new environment :smile: thanks Gunnar!

Reply:
You're welcome! I guess you wouldn't notice this when you usually launch the game through the taskbar menu. :)